rules-engine/tests/test_rules_engine/test_examples.py

```diff
@@ -19,69 +26,14 @@
 INPUT_DATA = filter(lambda d: d != "example-2", next(os.walk(ROOT_DIR))[1])
 
 
-def validate_fuel_type(value):
-    try:
-        return engine.FuelType[value]
-    except KeyError as e:
-        raise ValueError(
-            f"Error validating fuel type {e}. Valid choices are: {[x.name for x in engine.FuelType]}"
-        )
-
-
-class Summary(BaseModel):
+class Summary(SummaryInput, SummaryOutput):
     local_weather_station: str
-    design_temperature_override: Optional[float]
-    living_area: int
-    fuel_type: Annotated[engine.FuelType, BeforeValidator(validate_fuel_type)]
-    heating_system_efficiency: float
-    other_fuel_usage: Optional[float]
-    other_fuel_usage_override: Optional[float]
-    thermostat_set_point: float
-    setback_temp: Optional[float]
-    setback_hours_per_day: Optional[float]
-    estimated_balance_point: float
-    balance_point_sensitivity: float
-    average_indoor_temperature: float
-    difference_between_ti_and_tbp: float
-    design_temperature: float
-    whole_home_ua: int
-    standard_deviation_of_ua: float
-    avg_heat_load: int
-    max_heat_load: int
-
-
-def validate_usage_date(value):
```

Comment:
Did we lose some validation here, or has this been superseded in the pydantic models?